### PR TITLE
update README to explain the future of next on netlify

### PR DESCRIPTION
**Pull request by lindsaylevine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,22 @@
...
+
+See the [plugin's README](https://github.com/netlify/netlify-plugin-nextjs#readme) for more information. Our team's end goal is to make the Next.js on Netlify experience as seamless as possible.
+
+Note: since `next-on-netlify` will remain the primary source of logic for Next.js on Netlify, the existing `next-on-netlify` setup will continue to work for users who prefer manual installation and configuration.
+
+The plugin can be found on [npm here](https://www.npmjs.com/package/@netlify/plugin-nextjs).
+
+## About
 
 `next-on-netlify` is a utility for enabling server-side rendering in Next.js on Netlify. It wraps your application in a tiny compatibility layer, so that pages can use Netlify Functions to be server-side rendered.
```
Comment (sdras) [inline README.md:35]: This still need to be at the top- people need to know what they're looking at with a sentence or two before anything else. 

We also two links in bullets right after this sentence before moving on:
- build plugin link
- npm package

File: README.md [hunk 2]
```diff
@@ -15,6 +15,22 @@
   </a>
 </p>
 
+## The Future of Next.js on Netlify
```
Comment (sdras) [inline README.md:18]: I'd prefer wording like "Next on Netlify Roadmap" here.

**Review by sdras — CHANGES_REQUESTED** [review]
Looks good! Just a few small changes here.

**Review by sdras — APPROVED** [review]
Looks great!

**Comment by ChrisBAshton** [comment]
Hi 👋 the future looks exciting!

But I think we could do with highlighting that `@netlify/plugin-nextjs` is [not yet ready for use with Netlify CLI](https://github.com/netlify/netlify-plugin-nextjs#cli). I just tried migrating from one to the other before I read the note tucked away in the caveats section! It looks as though `next-on-netlify` is still the recommended approach if you're using the CLI.

**Comment by lindsaylevine** [comment]
@ChrisBAshton hey chris! thanks so much for calling this out, i agree this was an oversight. my thoughts at the moment:

1. [this](https://github.com/netlify/netlify-plugin-nextjs/issues/58#issuecomment-730439105) is the real issue using the plugin with the CLI, for full transparency. basically, re-runs of `netlify build` or `netlify deploy --build` will fail if you don't get rid of the functions and publish dirs that the plugin generates. reason: we took out the "clean directory" logic for the plugin to support custom functions, but of course, that created the pain points laid out in that issue. all that said, it is _possible_ to use the CLI with the plugin; it just means you have to do the manual clean yourself at the moment. just sayin'

2. that said, since this was a reason for you to _not_ switch, i'm going to prioritize a fix for it. i'll tag you on the issue (and then the PR/release) to keep you updated.

thank you again!!!!!!

**Pull request merged**